hal/halx86/xbox/halxbox.h

```diff
@@ -1,11 +1,9 @@
 /*
- * COPYRIGHT:       See COPYING in the top level directory
  * PROJECT:         Xbox HAL
- * FILE:            hal/halx86/xbox/halxbox.h
+ * LICENSE:         GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
```

Comment:
This should be `GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)`
But I am not sure that the `COPYING` file was unmodified GPL2+?

Reply:
I'm not sure too, but ok. :sweat_smile: